kpatch-build/kpatch-elf.c

```diff
@@ -858,6 +860,8 @@ void kpatch_elf_teardown(struct kpatch_elf *kelf)
 	}
 
 	list_for_each_entry_safe(sym, safesym, &kelf->symbols, list) {
+		if (sym->twin)
+			sym->twin = NULL;
```

Comment:
Just curious, where do the stale references occur?  
In this instance, wouldn't the subsequent memset be clearing sym->twin value anyway?

Reply:
Whoops, typo from my end, this should be like for the sections:
`sym->twin->twin = NULL;`

I don't think there is currently a bug due to that. It is just to make sure no-one is tempted to check the `twin` pointers of the patched object when building the final object as they reference freed object.